api/gluster-create-volume-example.adoc

```diff
@@ -0,0 +1,131 @@
+// vim: tw=79
+= Gluster Volume
+
+The document describes the flow for creating a new Gluster volume using the
+Tendrl API.
+
+To begin with, we need to explore the available actions that you can take on a
+volume, to discover the actions we can use the actions API on the
+resource for the defined cluster.
+
+Sample Request
+
+----------
+GET /cluster/:cluster_id/volume/actions
```

Comment:
How do you differentiate between Gluster cluster and Ceph cluster?
Will similar Ceph functions look like this:
```
GET /cluster/:cluster_id/pool/actions
GET /cluster/:cluster_id/pool/attributes
```
I think there should be obvious that it is call about Ceph or Gluster cluster or it should be unified, for example like this:
```
GET /cluster/:cluster_id/actions
GET /cluster/:cluster_id/attributes
```
where `:cluster_id` is Ceph or Gluster cluster id.